lib/evss/auth_headers.rb

```diff
@@ -37,11 +37,23 @@ def eauth_json
           edi: @user.edipi,
           firstName: @user.first_name,
           lastName: @user.last_name,
-          birthDate: iso8601_birth_date
+          birthDate: iso8601_birth_date,
+          gender: gender
         }
       }.to_json
     end
 
+    def gender
+      case @user.gender
+      when 'F'
+        'FEMALE'
+      when 'M'
+        'MALE'
+      else
+        ''
```

Comment:
Might such a raise come from [a new policy check](https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/policies/evss_policy.rb)? Or is that an incorrect use of Pundit policies? e.g.
```ruby
  def access_form526?
    user.gender.present?
  end
```

Reply:
@omgitsbillryan 🤔 @hpjaj had a similar issue, and we came up with a rule that: 

> If a required user attribute is nil, but it's unexpected, then it's and an error. If a required attribute is nil, but it's expected that some users can be in that state, then it can be part of a policy.

In this case we expect M or F and never expect nil or blank so I lean towards an error.